Makefile

```diff
@@ -209,6 +209,7 @@ watch-browser-node: browser-node
 	bash ops/test-unit.sh browser-node watch
 
 test-router: router-js
+	bash ops/start-messaging.sh
```

Comment:
Should we really use real messaging here? Why not the MemoryMessagingService?

Reply:
Memory messaging service is not really used anywhere, and many of the methods just throw the `Method not implemented` error.

I just copied the setup for the server-node tests which rely on a messaging service being created